### ISSUE TITLE
Any plan for supporting other platform?

### ISSUE THREAD

**Issue by fredgan** [ossf/criticality_score#361]
Hi, folks.

As far as I know, now criticality_score only support GITHUB.

Do we have any plan for supporting other platforms, such as GITLAB, [GITEE](https://gitee.com/)

**Comment by mudongliang** [comment]
I also wonder about this issue, and even if we can support other platforms, some metrics cannot compare across platforms, e.g., stars.

**Comment by vpamrit** [comment]
True, I imagine there would need to be some kind of normalized score across platforms based on users. For independent platforms, like gnu maybe we can use something else.
